qa/python_models/custom_metrics/model.py

```diff
@@ -136,6 +226,27 @@ def test_gauge_e2e(self):
         metrics = self._get_metrics()
         self.assertIn(pattern, metrics)
 
+    def test_histogram_e2e(self):
+        name = "test_histogram_e2e"
+        metric_family = pb_utils.MetricFamily(
+            name=name,
+            description="test metric histogram kind end to end",
+            kind=pb_utils.MetricFamily.HISTOGRAM,
+        )
+        labels = {"example1": "counter_label1", "example2": "counter_label2"}
+        buckets = [0.1, 1.0, 2.5, 5.0, 10.0]
```

Comment:
Testing duplicate values will be useful later on as well:
```python
duplicate_buckets = [1, 1, 1, 2, 2, 2]
metric = metric_family.Metric(labels=labels, buckets=duplicate_buckets)
# assert there is only one of each bucket value in metrics output, or capture
# error/exception, depending on how it behaves today
```

Reply:
Updated